lua/minigames/minigames/minigame_base/shared.lua

```diff
@@ -58,4 +65,20 @@ if SERVER then
 	function MINIGAME:IsSelectable()
 		return true
 	end
+else
+	---
+	-- This function is called as soon as the @{MINIGAME} is activated and should be used to display a EPOP message informing the player about the activation
+	-- @realm client
+	function MINIGAME:ShowActivationEPOP()
+		if not istable(self.lang) or not EPOP then return end
+
+		EPOP:AddMessage({
+				text = LANG.TryTranslation("ttt2_minigames_" .. self.name .. "_name"),
+				color = COLOR_ORANGE
+			},
+			self.lang.desc and LANG.TryTranslation("ttt2_minigames_" .. self.name .. "_desc") or nil,
+			12,
+			true
```

Comment:
What purpose does this `true` serve?

When I tested this, I was getting the following error:

[TTT2 (Base) - v0.7.3b - New Roles, Extensible, Better Performance, ULX Support] gamemodes/terrortown/gamemode/shared/hud_elements/ttteventpopup/pure_skin_eventpopup.lua:102: attempt to get length of field 'iconTable' (a boolean value)
  1. PrepareItem - gamemodes/terrortown/gamemode/shared/hud_elements/ttteventpopup/pure_skin_eventpopup.lua:102
   2. Draw - gamemodes/terrortown/gamemode/shared/hud_elements/ttteventpopup/pure_skin_eventpopup.lua:122
    3. DrawElemAndChildren - gamemodes/terrortown/gamemode/shared/huds/base_huds/hud_base/cl_init.lua:304
     4. Draw - gamemodes/terrortown/gamemode/shared/huds/base_huds/hud_base/cl_init.lua:330
      5. DrawHUD - gamemodes/terrortown/gamemode/client/cl_hud_manager.lua:57
       6. unknown - gamemodes/terrortown/gamemode/client/cl_hud_manager.lua:97

But if I remove this `true` it no longer gives an error.

Reply:
@Histalek https://github.com/TTT-2/ttt2mg/pull/5

Reply:
> What purpose does this `true` serve?
> 
> When I tested this, I was getting the following error:
> 
> [TTT2 (Base) - v0.7.3b - New Roles, Extensible, Better Performance, ULX Support] gamemodes/terrortown/gamemode/shared/hud_elements/ttteventpopup/pure_skin_eventpopup.lua:102: attempt to get length of field 'iconTable' (a boolean value)
> 
>     1. PrepareItem - gamemodes/terrortown/gamemode/shared/hud_elements/ttteventpopup/pure_skin_eventpopup.lua:102
> 
>     2. Draw - gamemodes/terrortown/gamemode/shared/hud_elements/ttteventpopup/pure_skin_eventpopup.lua:122
>        3. DrawElemAndChildren - gamemodes/terrortown/gamemode/shared/huds/base_huds/hud_base/cl_init.lua:304
>        4. Draw - gamemodes/terrortown/gamemode/shared/huds/base_huds/hud_base/cl_init.lua:330
>        5. DrawHUD - gamemodes/terrortown/gamemode/client/cl_hud_manager.lua:57
>        6. unknown - gamemodes/terrortown/gamemode/client/cl_hud_manager.lua:97
> 
> 
> But if I remove this `true` it no longer gives an error.

The true makes the epop message blocking, meaning it stays on screen for the full duration and can't be overwritten by another epop message. 

there seems to be missing an argument between the 12 and the true, `nil` should work i think as the icontable is optional